components/AvalancheCenterSelector.tsx

```diff
@@ -43,56 +44,88 @@ const center_idsByType: SectionListData<string>[] = [
 ];
 
 interface AvalancheCenterCardProps {
-  center_id: string;
-  set_avalanche_center: React.Dispatch<React.SetStateAction<string>>;
+  avalancheCenterId: string;
+  selected: boolean;
+  onPress: (avalancheCenter: string) => void;
 }
 
-const AvalancheCenterCard: React.FunctionComponent<AvalancheCenterCardProps> = ({center_id, set_avalanche_center}: AvalancheCenterCardProps) => {
-  const navigation = useNavigation<TabNavigationProps>();
-  const {isLoading, isError, data: avalancheCenter, error} = useAvalancheCenterMetadata(center_id);
+export const AvalancheCenterCard: React.FunctionComponent<AvalancheCenterCardProps> = ({avalancheCenterId, selected, onPress}: AvalancheCenterCardProps) => {
+  const {width} = useWindowDimensions();
+  const {isLoading, isError, data: avalancheCenter, error} = useAvalancheCenterMetadata(avalancheCenterId);
   if (isLoading) {
     return <ActivityIndicator style={styles.item} />;
   }
   if (isError) {
     return (
-      <View style={styles.item}>
-        <Text>{`Could not fetch data for ${center_id}: ${error?.message}.`}</Text>
-      </View>
+      <Box>
+        <Box bg="light.100">
```

Comment:
for styling things such as this, would it make sense to maybe centralize them somewhere? applies to Box (selected or not), NBText, HStack, etc - maybe through a common style sheet or something like that?

Reply:
Yeah, I think Brian's working on something like that - the styling/visual bits in this PR were copy-pasted from the proto-work we already have, and I'm going to come by and clean things up once we have a better approach figured out.